src/components/block/index.ts

```diff
@@ -520,6 +520,30 @@ export default class Block {
     }
   }
 
+  /**
+   * Clone Block's data
+   * Groups Tool's clone processing time
+   *
+   * @returns {object}
+   */
+  public async clone(): Promise<void | SavedData> {
+    try {
+      const measuringStart = window.performance.now();
+      const extractedBlock = await this.tool.save(this.pluginsContent as HTMLElement);
+      const clonedBlock = this.tool.clone ? (await this.tool.clone(extractedBlock)) : extractedBlock;
```

Comment:
`clone()` interface is option. 
If `clone()` is not implemented, use `save()` by default.